.github/requirements.txt

```diff
@@ -7,4 +7,5 @@ google-cloud-container
 google-cloud-tpu>=1.16.0
 jsonlines
 tensorflow-cpu
-apache-airflow-providers-cncf-kubernetes
\ No newline at end of file
+apache-airflow-providers-cncf-kubernetes
+kubernetes
```

Comment:
Is this needed for terraform? If not, could we add something like this? https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/deployment/cloud_composer_template.tf#L143

Reply:
Good point. This package is preinstalled, so I added a commented dep.